src/plugins/visualizations/public/wizard/group_selection/group_selection.tsx

```diff
@@ -0,0 +1,257 @@
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import React from 'react';
+import { orderBy } from 'lodash';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCard,
+  EuiIcon,
+  EuiModalHeader,
+  EuiModalBody,
+  EuiModalHeaderTitle,
+  EuiLink,
+  EuiText,
+  EuiSpacer,
+  EuiBetaBadge,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { DocLinksStart } from '../../../../../core/public';
+import { VisTypeAlias } from '../../vis_types/vis_type_alias_registry';
+import type { VisType, TypesStart } from '../../vis_types';
+import { VisGroups } from '../../vis_types';
+
+interface GroupSelectionProps {
+  onVisTypeSelected: (visType: VisType | VisTypeAlias) => void;
+  visTypesRegistry: TypesStart;
+  docLinks: DocLinksStart;
+  toggleGroups: (flag: boolean) => void;
+  showExperimental: boolean;
+}
+
+interface VisCardProps {
+  onVisTypeSelected: (visType: VisType | VisTypeAlias) => void;
+  visType: VisType | VisTypeAlias;
+  showExperimental?: boolean | undefined;
+}
+
+function isVisTypeAlias(type: VisType | VisTypeAlias): type is VisTypeAlias {
+  return 'aliasPath' in type;
+}
+
+function GroupSelection(props: GroupSelectionProps) {
+  const visualizeGuideLink = props.docLinks.links.dashboard.guide;
+  const promotedVisGroups = orderBy(
+    [
+      ...props.visTypesRegistry.getAliases(),
+      ...props.visTypesRegistry.getByGroup(VisGroups.PROMOTED),
+    ],
+    ['promotion', 'title'],
+    ['asc', 'asc']
+  );
+  return (
+    <>
+      <EuiModalHeader>
+        <EuiModalHeaderTitle data-test-subj="groupModalHeader">
+          <FormattedMessage
+            id="visualizations.newVisWizard.title"
+            defaultMessage="New Visualization"
+          />
+        </EuiModalHeaderTitle>
+      </EuiModalHeader>
+      <EuiModalBody>
+        <EuiSpacer size="s" />
+        <EuiFlexGroup
+          gutterSize="l"
+          wrap
+          justifyContent="center"
+          responsive={false}
+          data-test-subj="visNewDialogGroups"
+        >
+          {promotedVisGroups.map((visType) => (
+            <VisGroup
+              visType={visType}
+              key={visType.name}
+              onVisTypeSelected={props.onVisTypeSelected}
+            />
+          ))}
+        </EuiFlexGroup>
+        <EuiSpacer size="xl" />
```

Comment:
I want to create the gap between the promoted Groups and the AggBased, Tools and for this reason I have used two xl EuiSpacer. Is there another way to do it? cc @miukimiu